primitives/xcm/src/lib.rs

```diff
@@ -360,21 +337,3 @@ impl<
 		}
 	}
 }
-
-// Multi IsConcrete Implementation. Allows us to route both pre and post 0.9.16 anchoring versions
-// of our native token to the same currency
-// The incoming MultiAsset is matched against a Vec of multilocations and returned Some
-// if matches
-pub struct MultiIsConcrete<T>(PhantomData<T>);
```

Comment:
Not needed anymore, since we no longer accept two versions of our SelfReserve token.